### DISCUSSION TITLE
v2.0.6 播放器投影

### DISCUSSION THREAD

**Discussion by FantasticAiming** [the1812/Bilibili-Evolved#2635] [功能建议 / Ideas]
从正式版转过来 感觉播放器投影范围变小了 颜色似乎也变浅了 这样看着不太明显  
所以希望作者能增加一个自定义投影范围以及深浅度的功能
另外 网页宽屏模式跟正式版比起来变小了挺多  不知道是最近改版引起的还是原来就是这样  

噢..还有一个小建议 就是增加一个快捷键开启夜间模式的功能

**Comment by Geeyun-JY3** [comment]
#1297
先前提过的而且都已经打过 tag 的建议就不用再复读一遍了：#2556

**Reply by FantasticAiming** [reply]
好的 不常用github  忘记了...
